src/Common/CurrentMetrics.cpp

```diff
@@ -70,6 +70,7 @@
     M(ZooKeeperSession, "Number of sessions (connections) to ZooKeeper. Should be no more than one, because using more than one connection to ZooKeeper may lead to bugs due to lack of linearizability (stale reads) that ZooKeeper consistency model allows.") \
     M(ZooKeeperWatch, "Number of watches (event subscriptions) in ZooKeeper.") \
     M(ZooKeeperRequest, "Number of requests to ZooKeeper in fly.") \
+    M(ZooKeeperExpiredSessions, "Number of expired ZooKeeper sessions. Includes only non-auxiliary sessions managed by the context.") \
```

Comment:
Probably nitpicking, but imo `ZookeeperSessionExpired` counter - looks more suitable. In case we'd want to do a separate metrics for auxiliary and non-auxiliary, - `ZookeeperAuxiliarySessionExpired` can be used for auxiliary sessions